src/CacheCow.Server/CachingHandler.cs

```diff
@@ -1,646 +1,654 @@
-﻿using System;
```

Comment:
I am sorry but the diff is showing the whole of the file, can you please point out what was the line you changed?